docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -33,7 +33,8 @@ kyma alpha deploy [flags]
       --value strings             Set a configuration value (e.g. --value component.key='the value').
   -f, --values-file strings       Path to a JSON or YAML file with configuration values.
       --workers-count int         Number of parallel workers used for the deployment. (default 4)
-  -w, --workspace string          Path used to download Kyma sources. (default "workspace")
+  -w, --workspace string          If --source is local, then workspace should be a path already containing Kyma sources.
+                                  If --source is not local, then workspace would be the path used to download Kyma sources. (default "workspace")
```

Comment:
```suggestion
                                  If --source is not local, then workspace must be the path used to download Kyma sources (default: "workspace").
```

Reply:
the part for (default "workspace") is auto-generated and we should not change it manually

Reply:
I will change "would" to "must" as suggested, but the other changes should not be applied

Reply:
Alright! I didn't know that this is auto-generated. 
I'm not a fan of how it is rendered tho. It's better with a colon and in front of the full stop, but no need to change it here.